### PR TITLE
home 1 y home 2 con cabecera de las demas paginas como componente

### PR DESCRIPTION
**Pull request by lekyam**

### CODE REVIEW

File: PimProject/PimWebApp/AuthController.cs [hunk 1]
```diff
@@ -22,5 +22,8 @@ public async Task<ActionResult> Google()
             };
             return Challenge(properties, GoogleDefaults.AuthenticationScheme);
         }
+
```
Comment (utarefson) [inline PimProject/PimWebApp/AuthController.cs:25]: Please undo this file.

Comment (lekyam) [reply PimProject/PimWebApp/AuthController.cs:25]: eso es del Gabo ing

File: PimProject/PimWebApp/Shared/SurveyPrompt.razor [hunk 2]
```diff
@@ -13,4 +13,5 @@
     // Demonstrates how a parent component can supply parameters
     [Parameter]
     public string Title { get; set; }
+
```
Comment (utarefson) [inline PimProject/PimWebApp/Shared/SurveyPrompt.razor:16]: Please undo this file.

File: PimProject/PimWebApp/PimWebApp.csproj [hunk 3]
```diff
@@ -8,4 +8,6 @@
     <PackageReference Include="Microsoft.AspNetCore.Authentication.Google" Version="3.1.8" />
   </ItemGroup>
 
+  <ProjectExtensions><VisualStudio><UserProperties appsettings_1json__JsonSchema="" /></VisualStudio></ProjectExtensions>
```
Comment (utarefson) [inline PimProject/PimWebApp/PimWebApp.csproj:11]: Do you need this ProjectExtensions?

**Review by utarefson — CHANGES_REQUESTED** [review]
Please take a look at my comments.

**Comment by utarefson** [comment]
.

**Review by utarefson — APPROVED** [review]

**Pull request merged**